packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -171,8 +167,42 @@ export const jiraIssue = (
                 })
               })
             )
+
+            const VALID_TYPES = ['string', 'number']
+            const INVALID_WORDS = ['color', 'name', 'description', 'environment']
+            const INVALID_NOT_SIMPLIFIED_FIELD = 'story point estimate'
+            const INVALID_SIMPLIFIED_FIELD = 'story points'
```

Comment:
i wonder if these constants belong in a separate file rather than inlining them here